DispatchR/Dispatcher.cs

```diff
@@ -94,20 +101,22 @@ public Task RunAsync(CancellationToken token = default)
             }, token);
         }
 
-        protected void Dispatch(List<Dispatchee> dispatchees, CancellationToken token = default)
+        protected static void Dispatch(List<Dispatchee> dispatchees, CancellationToken token = default)
         {
-            lock (_lock)
+            Dispatchee[] array;
```

Comment:
Allocating new memory indefinitely in a while loop?

Reply:
It will get garbage collected in a background thread... Not ideal, but safe considering the list collection could be modified.  Maybe the lock should be on the `SyncRoot` of the `List`.